zuul.d/tcib.yaml

```diff
@@ -0,0 +1,19 @@
+---
+- job:
+    name: cifmw-base-tcib
```

Comment:
Please add this job `cifmw-base-tcib` to https://github.com/openstack-k8s-operators/ci-framework/blob/main/zuul.d/projects.yaml#L8